src/components/ProjectList/ProjectContext.js

```diff
@@ -0,0 +1,48 @@
+import React, { createContext, useEffect, useState } from 'react';
+import { API } from '../../store/api';
+import PropTypes from 'prop-types';
+
+export const ProjectsContext = createContext();
+
+const ProjectsContextProvider = (props) => {
+  const [projects, setProjects] = useState([]);
+  const [activeProjects, setActiveProjects] = useState(true);
+  const [reload, setReload] = useState(true);
+  useEffect(() => {
```

Comment:
You can change it to 
```
  useEffect(() => {
    API.get(`/autographa/projects?active=${activeProjects}&limit=100`)
      .then((response) => {
        setProjects(response.data);
        setReload(false);
      })
      .catch((err) => console.log(err));
  }, [reload, setReload, activeProjects]);
```
This will take care of both cases so no need for the next useEffect